test/test_cli.py

```diff
@@ -154,9 +154,6 @@ def test_invalid_parameter(self):
         assert e.value.args[0] == ERROR_CODES[SystemExit]
 
     @pytest.mark.skipif(not LONG_TESTS(), reason="Update flag tests are long tests")
```

Comment:
I'm glad to see these can be re-enabled on windows, but while we're doing the changes, let's think about this one. test_update_flags currently tests `-u now` which forces a full database refresh.

Two options
1. Move this test to the external tests (e.g. check for EXTERNAL() ) 
2. Remove the `-u now` part of the check entirely. 

I think we should go with the latter: we can replace the check when we switch from using NVD directly to using our own cache, but in the meantime we know the -u now code is run daily so it's not like it's not being tested, it just won't be tested on a per-commit basis in exchange for faster CI runs and more stable results from said CI runs.

Reply:
> but in the meantime we know the -u now code is run daily (in the cache update job) so it's not like it's not being tested

BTW 😮
![image](https://user-images.githubusercontent.com/12860284/210454381-3fd8d477-9847-4ea5-8ec8-d3549608cd98.png)